### DISCUSSION TITLE
My auto-icrement IDs in MySQL get incremented even if a create was unsuccessful

### DISCUSSION THREAD

**Discussion by dbalatoni13** [prisma/prisma#6387] [Q&A] [answered]
Is it normal that my auto-icrement IDs in MySQL get incremented even if a create was unsuccessful and threw an exception? What can I do against this?

**Comment by janpio** [comment, marked as answer]
Sounds like this is common MySQL behavior: https://stackoverflow.com/a/17799060

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
